angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html

```diff
@@ -29,25 +29,28 @@ <h4 id="appTitle_portlet.title-{{::portlet.nodeId}}" aria-labelledby="appTitle_p
     </div>
     <a class="btn btn-default launch-app-button" href="{{::portlet.url}}" target="{{::portlet.target}}">Launch full app</a>
   </div>
-  
+
   <div ng-if="'OPTION_LINK' === widgetCtrl.portletType(portlet)">
     <option-link app="portlet" config="portlet.widgetConfig"></option-link>
   </div>
-  
+
   <div ng-if="'WEATHER' === widgetCtrl.portletType(portlet)">
     <weather app="portlet" config="portlet.widgetConfig"></weather>
   </div>
-  
+
   <div ng-if="'RSS' === widgetCtrl.portletType(portlet)">
     <rss app="portlet" config="portlet.widgetConfig"></rss>
   </div>
-  
+
   <div ng-if="'GENERIC' === widgetCtrl.portletType(portlet)">
     <div ng-controller="GenericWidgetController as genericWidgetCtrl">
+        <div ng-if="loading" id="loading">
+            <loading-gif data-object='content' data-empty='isEmpty'></loading-gif>
+        </div>
         <content-item></content-item>
```

Comment:
I think we need it, when I try to remove it and run, the loading-gif will
be there forever.

On Tue, Oct 13, 2015 at 9:02 AM, Tim Levett <notifications@github.com>
wrote:

> In
> angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html
> <https://github.com/UW-Madison-DoIT/angularjs-portal/pull/333#discussion_r41868640>
> :
>
> >    <div ng-if="'GENERIC' === widgetCtrl.portletType(portlet)">
> >      <div ng-controller="GenericWidgetController as genericWidgetCtrl">
> > +        <div ng-if="loading" id="loading">
> > +            <loading-gif data-object='content' data-empty='isEmpty'></loading-gif>
> > +        </div>
> >          <content-item></content-item>
>
> Should we add in a ng-if='!loading' here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UW-Madison-DoIT/angularjs-portal/pull/333/files#r41868640>
> .
>